core/store/src/test/java/uk/gov/gchq/gaffer/store/operation/OperationChainValidatorTest.java

```diff
@@ -49,6 +53,18 @@
 import static org.mockito.Mockito.verify;
 
 public class OperationChainValidatorTest {
+    private final ViewValidator viewValidator = mock(ViewValidator.class);
+    private final OperationChainValidator validator = new OperationChainValidator(viewValidator);
+    private final Store store = mock(Store.class);
+    private final User user = mock(User.class);
+    private final Schema schema = mock(Schema.class);
+
+    @BeforeEach
+    public void setup() throws OperationException {
+        given(store.getSchema()).willReturn(schema);
+        given(store.execute(any(GetTraits.class), any(Context.class))).willReturn(new HashSet<>());
```

Comment:
So no traits are returned from mock?

Reply:
Yes it returns an empty set of traits, i.e. no traits. This was what happened before, so far as I can tell. We just have to mock it now as otherwise we get `null` from the operation.